.travis.yml

```diff
@@ -3,8 +3,9 @@ python:
     - 3.4
     - 3.3
     - 2.7
-sudo: false
+sudo: true
```

Comment:
Not sure if this will work, but you might be able to do:

```
addons:
  apt:
    packages:
    - pdflatex
    - texlive-base
    - pandoc
```

And then you wouldn't need to use sudo.